modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -0,0 +1,170 @@
+[[clients-sleses]]
+= Registering {sleses} Clients
+
+This section contains information about registering traditional and Salt clients running {sleses} operating systems ({slesesa} for the rest of the guide)
+
+{slesesa} clients can be initally based {rhel} or {centos}.
+
+[WARNING]
+====
+{slesesa} clients are also called sometimes SLESES, RES or {redhat} {slesesa})
+For {rhel} based clients tou are responsible for arranging access to {rhela} or {centos} base media repositories installation media.
```

Comment:
```suggestion
For {rhel} based clients you are responsible for arranging access to {rhela} or {centos} base media repository installation media.
```
Maybe, we can just say "installation media" instead of "base media repository installation media".